proc_Hahn_echoph.py

```diff
@@ -1,67 +1,45 @@
 from pyspecdata import *
 from scipy.optimize import leastsq,minimize,basinhopping
-fl = figlist_var()
+from proc_scripts import *
+from proc_scripts.load_data import postproc_lookup
+from sympy import symbols
+fl = fl_mod()
```

Comment:
OK, so fl_mod is defined in the package, and provides the new plotting methods so you can use them here?

Reply:
yes